### PR TITLE
Feature(Issue #7):  Create a welcoming prompt when the shopping list is empty

### LINKED ISSUE #7: 7. As a user, I want to see a welcoming prompt to add my first item if my list is empty to help me get oriented to how the app works.
AC:

* The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item

### PR DESCRIPTION
**Pull request by brewswain**
## Description

This Pull Request shows the user a welcoming prompt that asks them to add an Item by clicking the provided button.

This modification takes place in the `<Listener />` component.

## Related Issue

Closes issue #7

## Acceptance Criteria

- The list view, when there are no items to display, should show a prompt (e.g. a button) for the user to add their first item

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
| ✓   | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### Before

image

### After

image

## Testing Steps / QA Criteria

- From your terminal, run git checkout `bl-kl-welcome-prompt` branch
- Once once checked out to the `bl-kl-welcome-prompt` branch, in the browser, visit the new page: (http://localhost:3000 or as indicated in your terminal).
- Click the `List` link in the `Footer` below.
- You should see a `<Card />` component that informs you that your shopping list is empty, and ask for you to add an item.

### CODE REVIEW

File: src/components/Card/Card.component.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import './Card.style.scss';
+const Card = () => (
+  <div className="card__container">
+    <div className="text__container">
+      <h3 className="card__title">
+        Your shopping list is empty 
+        <span role="img" aria-label="A crying emoji" className="card__emoji">
```
Comment (joeylaguna) [inline src/components/Card/Card.component.jsx:9]: Nice aria-label here! :D

**Review by daredtech — APPROVED** [review]
Loved the prompt and the use of emojis :) Great job!

**Comment by joeylaguna** [comment]
Looks like there are some merge conflicts here. Once you resolve these conflicts I'll approve and you can merge 👍

**Review by jendevelops — APPROVED** [review]
Great job! Sorry for the late review!

**Comment by CodeMeKathy** [comment]
@brewswain, did you mean to `Merge branch 'main' into bl-kl-welcome-prompt`?

**Comment by brewswain** [comment]
> @brewswain, did you mean to `Merge branch 'main' into bl-kl-welcome-prompt`?

Hmm, I think that happened when I resolved a merge conflict since this was the extent of files changed.:
image

**Comment by daredtech** [comment]
I just noticed, when there are no tokens (empty local storage) and the user clicks `Create a New Shopping List`  Your shopping list is currently empty is displayed instead of `Card` component.

**Pull request merged**